src/helpers/getCatchError.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * @param {unknown} error Error returned in `try catch`
+ * @returns {Error}
+ */
+
+export const getCatchError = (error: unknown): Error => {
+  if (error instanceof Error) {
+    return error;
+  }
+
+  if (typeof error === "string" || typeof error === "number") {
+    return Error(error.toString());
+  }
+
+  return Error("Something went wrong, please try again.");

```

Comment:
Cann we actually change this default error to something like "An error was unable to be parsed correctly"? It'd be helpful to know that something went wrong trying to decode the error instead of a service failing